root/report/KaraokePlusInstrumentalRecordings.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import RecordingList from './components/RecordingList.js';
+import ReportLayout from './components/ReportLayout.js';
+import type {ReportDataT, ReportRecordingT} from './types.js';
+
+component KaraokePlusInstrumentalRecordings(...{
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportRecordingT>) {
+  return (
+    <ReportLayout
+      canBeFiltered={canBeFiltered}
+      description={exp.l(
+        `This report shows recordings which have a
+         {doc_karaoke_rel|“karaoke” relationship} to another recording,
+         but are linked to a work with the
+         {doc_instrumental_attr|“instrumental” attribute}, or the other
+         way around, with a
+         {doc_instrumental_rel|“instrumental” relationship} and a
+         {doc_karaoke_attr|“karaoke” attribute}.`,
+        {
+          doc_instrumental_attr: '/relationship-attributes#instrumental',
+          doc_instrumental_rel:
+            '/relationship/9fc01a58-7801-4bd2-b07d-61cc7ffacf90',
+          doc_karaoke_attr: '/relationship-attributes#karaoke',
+          doc_karaoke_rel:
+            '/relationship/39a08d0e-26e4-44fb-ae19-906f5fe9435d',
+        },
+      )}
+      entityType="relationship"
+      extraInfo={l(
+        `Keep in mind that “instrumental” in MusicBrainz implies the lyrics
+         are not relevant to this recording. Since lyrics are by definition
```

Comment:
(github ate this comment when I submitted the previous review somehow?)

"the" sounds a bit better to me here since it's talking about any instrumental recording, not a specific one
```suggestion
         are not relevant to the recording. Since lyrics are by definition
```